TYPO3.Flow/Classes/TYPO3/Flow/Core/Booting/Scripts.php

```diff
@@ -161,10 +178,16 @@ public static function initializeSignalSlot(Bootstrap $bootstrap)
      */
     public static function initializePackageManagement(Bootstrap $bootstrap)
     {
-        $packageManager = new \TYPO3\Flow\Package\PackageManager();
-        $bootstrap->setEarlyInstance(\TYPO3\Flow\Package\PackageManagerInterface::class, $packageManager);
+        $packageManager = new PackageManager();
+        $bootstrap->setEarlyInstance(PackageManagerInterface::class, $packageManager);
+
+        // The package:rescan must happen as early as possible, compiletime alone is not enough.
```

Comment:
The next three lines contain the actual bugfix, the rest is just use statements.